tests/integration/src/test/resources/org/mule/test/integration/filter/regex-test.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="
+							http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+							">
+
+	<flow name="regex-filter-default-payload-positive">
+        <set-payload value="#['run with the mules']"/>

```

Comment:
You can avoid the almost duplicate flows setting the payload in the tests. That will also make the tests clearer.